app/views/schools/advice/electricity_out_of_hours/_analysis.html.erb

```diff
@@ -0,0 +1,47 @@
+<p><%= t('advice_pages.electricity_out_of_hours.analysis.summary') %></p>
+<ul>
+  <li>
+    <a href='#last_twelve_months'>
+      <%= t('advice_pages.electricity_out_of_hours.analysis.last_twelve_months.title') %>
+    </a>
+  </li>
+  <li>
+    <a href='#usage_by_day_of_week'>
+      <%= t('advice_pages.electricity_out_of_hours.analysis.usage_by_day_of_week.title') %>
+    </a>
+  </li>
+</ul>
+
+<%= render 'schools/advice/section_title', section_id: 'last_twelve_months', section_title: t('advice_pages.electricity_out_of_hours.analysis.last_twelve_months.title') %>
+<% if @school.school_times.community_use.any? %>
+  <p><%= t('advice_pages.electricity_out_of_hours.analysis.last_twelve_months.community_use') %></p>
+<% end %>
+
+<p><%= t('advice_pages.electricity_out_of_hours.analysis.last_twelve_months.table_introduction') %></p>
+
+<%= render 'analysis_table' %>
+
+<%= component 'chart', chart_type: :daytype_breakdown_electricity_tolerant, school: @school do |c| %>
+  <% c.with_title do %>
+    <%= t('advice_pages.electricity_out_of_hours.analysis.usage_by_day_of_week.daytype_breakdown_electricity_tolerant_chart.title') %>
+  <% end %>
+  <% c.with_subtitle do %>
+    <%= t('advice_pages.electricity_out_of_hours.analysis.usage_by_day_of_week.daytype_breakdown_electricity_tolerant_chart.subtitle') %>
```

Comment:
Can you please update the subtitles for both charts so that they have the start and end dates that they're showing, and wrap those in the <span class="start-date"></span> and end-date markup so that the dates are dynamic? This diverges from the document slightly, but I'm trying to standardise those across all pages.